app/components/work_packages/types/pattern_autocompleter.html.erb

```diff
@@ -0,0 +1,119 @@
+<%#-- copyright
+OpenProject is an open source project management software.
+Copyright (C) the OpenProject GmbH
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License version 3.
+
+OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+Copyright (C) 2006-2013 Jean-Philippe Lang
+Copyright (C) 2010-2013 the ChiliProject Team
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+See COPYRIGHT and LICENSE files for more details.
+
+++#%>
+
+<%=
+  content_tag(
+    :div,
+    class: "pattern-autocompleter",
+    "data-controller": "pattern-autocompleter",
+    "data-pattern-autocompleter-pattern-initial-value": @value
+  ) do
+%>
+  <%=
+    @input.builder.hidden_field(
+      name,
+      value: @value,
+      data: {
+        "pattern-autocompleter-target": "formInput"
+      }
+    )
+  %>
+
+  <template data-pattern-autocompleter-target="tokenTemplate">
+    <%=
+      render(
+        Primer::Beta::Label.new(
+          contenteditable: false,
+          tag: :span, display: :inline, scheme: :accent,
+          "data-role": :token
+        )
+      ) do
+    %>
+      <%= content_tag(:span, style: "cursor: default;", "data-role": "token-text") { "__VALUE__" } %>
```

Comment:
Please do  not use inline styles. Either use system_arguments or if the style is not supported, add a class and write the styles in the CSS file.